framework/source/class/qx/ui/table/IColumnMenuItem.js

```diff
@@ -37,5 +29,27 @@ qx.Interface.define("qx.ui.table.IColumnMenuItem",
      * item is now visible.
      */
     changeVisible : "qx.event.type.Data"
+  },
+
+  members :
+  {
+    /**
+     * Set whether the table column associated with this menu item is visible
+     *
+     * @param value {Boolean} whether the table column associated with this
+     *   menu item is visible
+     */
+    setVisible : function (value) {
+      this.assertBoolean(value);
+    },
+
+
+    /**
+     * Get whether the table column associated with this menu item is visible
+     *
+     * @return {Boolean} whether the table column associated with this menu
+     *   item is visible
+     */
+    getVisible : function () {}
   }
```

Comment:
Yes, but then we override `isVisible` and `toggleVisible`,too! The layout system seems to call `isVisible` on Widgets it likes to render, what causes strange effects (as far as I recognize).

"_mimic full boolean property_" is not something that is really needed for the Table classes to work.
Therefore the interface only demands to implement the `getVisible` and `setVisible` methods.

It might not be "_mimic full boolean property_", but this is the closest thing I could think of in terms of "**no API change**"
...I would still vote for a change in the API (Interface demanding `getColumnVisible` and `setColumnVisible` & Event), but it seems that API-change is ruled out :disappointed: